certified-connectors/SigningHub/Readme.md

```diff
@@ -0,0 +1,24 @@
+
+## SigningHub Connector
+SigningHub is a complete solution for document approval workflows, advanced digital signatures and document status tracking. It is designed to quickly optimize the way businesses deliver, review, approve and sign their business documents
+
+##Publisher
```

Comment:
There are still some errors with Readme.md file. ## Publisher is a correct way of defining header. Please review the file using this [template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/readme.md).